README.md

```diff
@@ -89,7 +89,7 @@ Some of the most common built-in matchers are
 - `s.object(fields)`
 - `s.objectWithOnly(fields)`
 - `s.regex(reg)`
-- `s.string()`
+- `s.string(min, max)`
```

Comment:
@nguyenchr Agree, do you think something like `s.string({min:min_length_value,max:max_length_value})` is good format to describe both accepted params?